[fix] Renamed site newpct -> descargas 2020 + changes

malkavi opened the pull request:
### Motivation for changes:
Newpct now is descargas2020 (or tvsinpagar)
### Detailed changes:
- Changed name to descargas2020

cvium commented on flexget/plugins/sites/descargas2020.py:
```diff
@@ -73,13 +66,22 @@ def parse_download_page(self, url):
                 if match:
                     torrent_id = match.group(2)
+                    if 'descargas2020.com' in url:
+                        return (torrent_id.replace('descargar-torrent','download') + '.torrent')
```
Two issues here.

1. There should be a space after the comma.
2. The outermost parenthesis is unnecessary.

cvium commented on flexget/plugins/sites/descargas2020.py:
```diff
@@ -73,13 +66,22 @@ def parse_download_page(self, url):
+                        return (torrent_id.replace('descargar-torrent','download') + '.torrent')
+                    else:
+                        response = task.requests.get(torrent_id)
+                        response = task.requests.get(response.url)
```
I don't like the re-use of variable here. Makes it unclear what's going on. It also _needs_ a try-except.

malkavi replied:
It's because there are redirects. but right now the feed does not work, maybe it's better to remove it for the moment. (It's the "else" for tvsinpagar, the feed is empty actualy)

Lukeid commented:
Hello @malkavi,

First of all, that you for you effort keeping updated and working this.

Just one suggestion, maybe you could include all realted pages:
   - descargas2020.com
   - tvsinpagar.com
   - tumejortorrent.com
   - torrentlocura.com
   - torrentrapid.com
 
Take a look at issue #2083.

Thank you again.

malkavi commented:
i have added the clones and fixed the issues. I have removed the code that isn't useful at this moment. All the clones are similar and use the same regexp to extract the torrent.

cvium commented:
Is this finished then?

malkavi commented:
It's working right now. It's finished, at least for me.

cvium commented:
Can you update the wiki?

malkavi commented:
Done. Search and urlrewriter updated, and removed newpct.
